workflows/quickstart/execute_workflow.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	executions "cloud.google.com/go/workflows/executions/apiv1beta"
+	executionspb "google.golang.org/genproto/googleapis/cloud/workflows/executions/v1beta"
+)
+
+// [START workflows_api_quickstart]
```

Comment:
Is this intended to be a quickstart or a snippet? Quickstarts should have a `func main`, meaning they can be run as a standalone binary. See https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#only-quickstarts-have-package-main.

I see you have a `func main` in another file -- that is ordinarily part of the sample. Otherwise, we try to avoid having that code at all.

Reply:
Quickstart.

Taken from https://github.com/GoogleCloudPlatform/golang-samples/tree/master/tasks

I can move the contents to main.